package.json

```diff
@@ -103,5 +103,8 @@
         "node-forge": "^0.10.0",
         "openrosa-xpath-evaluator": "^2.0.6",
         "signature_pad": "2.3.x"
+    },
+    "volta": {
+        "node": "12.22.1"
```

Comment:
Seems useful. Thanks. Just checking: do you know if Volta could use the information in the `"engines" : { "node": "12", "npm": "6"}` setting instead? That would also help non-Volta users.

Reply:
This intuitively seemed like something it'd support, but evidently they've [decided](https://github.com/volta-cli/volta/issues/355) against this because the fields have different semantics: `engines` specifies a range, while `volta` (née `toolchain`) pins a specific version.

Reply:
Thanks! Let's keep it then.